docs/admin.rst

```diff
@@ -513,6 +619,57 @@ codes. For example: ::
 You'll need to list all languages you now want to support, adding or removing
 languages as needed. Locale changes will be applied after the next reboot.
 
+Managing Configuration Updates with Multiple Admins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Organizations with multiple admins should set up a way to synchronize
+any changes one admin makes to the server configuration, as by default those
+changes are stored only on their individual *Admin Workstation*.
+
+Configuration changes will be flagged by OSSEC and will generate alerts, but
+if other admins don't regularly review OSSEC alerts they may miss important
+changes, such as an update to the *Submission Public Key*. If they subsequently
+run ``./securedrop-admin install`` from their *Admin Workstation*, they will
+revert the server configuration to an older version.
+
+The simplest approach to keeping workstations in sync is to inform other admins
+of changes as you make them, for example via a secure Signal group chat. Any such
+communications should happen over a platform that provides E2EE, as you may need to
+share sensitive information.
+
+Configuration information is stored in several files on the *Admin Workstation* under
+``~/Persistent/securedrop/``:
+
+* ``install_files/ansible-base/group_vars/all/site-specific`` contains settings written by
+  ``./securedrop-admin sdconfig`` - if it is changed other admins should be notified.
+* GPG public keys for the ``Submission Key`` and OSSEC Alert key should be present
```

Comment:
- *Submission Key* should be italicized (is preformatted)
- *OSSEC Alert Public Key* is also in the glossary now

Reply:
:white_check_mark: